src/controllers/contact.ts

```diff
@@ -1,12 +1,27 @@
 import { Request, Response } from 'express'
 import { check, validationResult } from 'express-validator'
 import asyncHandler from 'express-async-handler'
-import { NotImplemented, UnprocessableEntity } from '../errors'
+import { InternalServerError, UnprocessableEntity } from '../errors'
+import { logger } from '../logger'
+import { MailTemplate } from '../mailer/mailTemplate'
+import { mail } from '../mailer'
```

Comment:
This can be updated to pull in `MailTemplate` instead from the above import:
```suggestion
import { mail, MailTemplate } from '../mailer'
```

Reply:
Complete